db/model/Gdoc/Gdoc.ts

```diff
@@ -284,44 +282,51 @@ export class Gdoc extends BaseEntity implements OwidGdocInterface {
     ): Promise<void> {
         const slugToIdMap = await Chart.mapSlugsToIds()
         const uniqueSlugsByLinkType = this.links.reduce(
-            (slugsByLinkType, { linkType, target }) => {
+            (slugsByLinkType, { linkType, target, queryString }) => {
                 if (linkType === "grapher" || linkType === "explorer") {
-                    slugsByLinkType[linkType].add(target)
+                    slugsByLinkType[linkType].add({ target, queryString })
                 }
                 return slugsByLinkType
             },
-            { grapher: new Set<string>(), explorer: new Set<string>() }
+            {
+                grapher: new Set<{ target: string; queryString: string }>(),
+                explorer: new Set<{ target: string; queryString: string }>(),
+            }
         )
 
         const linkedGrapherCharts = await Promise.all(
-            [...uniqueSlugsByLinkType.grapher.values()].map(
-                async (originalSlug) => {
-                    const chartId = slugToIdMap[originalSlug]
-                    if (!chartId) return
-                    const chart = await Chart.findOneBy({ id: chartId })
-                    if (!chart) return
-                    const resolvedSlug = chart.config.slug ?? ""
-                    const resolvedTitle = chart.config.title ?? ""
-                    const linkedChart: LinkedChart = {
-                        originalSlug,
-                        title: resolvedTitle,
-                        resolvedUrl: `${BAKED_GRAPHER_URL}/${resolvedSlug}`,
-                        thumbnail: `${BAKED_GRAPHER_EXPORTS_BASE_URL}/${resolvedSlug}.svg`,
-                    }
-                    return linkedChart
+            [...uniqueSlugsByLinkType.grapher.values()].map(async (link) => {
+                const originalSlug = link.target
+                const queryString = link.queryString
+                const chartId = slugToIdMap[originalSlug]
+                if (!chartId) return
+                const chart = await Chart.findOneBy({ id: chartId })
+                if (!chart) return
+                const resolvedSlug = chart.config.slug ?? ""
+                const resolvedTitle = chart.config.title ?? ""
+                const linkedChart: LinkedChart = {
+                    originalSlug,
+                    queryString,
+                    title: resolvedTitle,
+                    resolvedUrl: `${BAKED_GRAPHER_URL}/${resolvedSlug}`,
```

Comment:
I don't know off the top of my head just now what `resolvedUrl` is used for, but should it include querystring when present maybe?